cdap-distributions/bin/.rubocop_todo.yml

```diff
@@ -0,0 +1,37 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-01-15 17:31:44 -0800 using RuboCop version 0.35.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+Lint/HandleExceptions:
+  Exclude:
+    - 'make_manifest.rb'
+
+# Offense count: 1
+Metrics/AbcSize:
+  Max: 32
+
+# Offense count: 2
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 107
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 35
+
+# Offense count: 1
+Style/AsciiComments:
+  Exclude:
+    - 'make_manifest.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/LeadingCommentSpace:
+  Exclude:
+    - 'make_manifest.rb'
```

Comment:
Should we move these files to the top of the repo and maintain them as repo global?